pkg/model/iam/subject.go

```diff
@@ -101,12 +101,10 @@ func BuildNodeRoleSubject(igRole kops.InstanceGroupRole, enableLifecycleHookPerm
 // ServiceAccountIssuer determines the issuer in the ServiceAccount JWTs
 func ServiceAccountIssuer(clusterSpec *kops.ClusterSpec) (string, error) {
 	if featureflag.PublicJWKS.Enabled() {
```

Comment:
Shouldn't this feature flag check be removed?

Reply:
This one got tricky to remove just like that. There is no flag that determines what the actual ServiceAccountIssuer is. It is coming in the next PR.